### PR TITLE
Feature/tailwind landing page

### PR DESCRIPTION
**Pull request by clari182**
Part of issue https://github.com/responsible-ai-collaborative/aiid/issues/604 :
Port landing page (https://incidentdatabase.ai/) to use Tailwind in the main content area

### CODE REVIEW

**Review by cesarvarela — CHANGES_REQUESTED** [review]
there is a weird margin/padding here: 

image

**Comment by clari182** [comment]
@cesarvarela fixed padding issues

**Comment by cesarvarela** [comment]
Conflicts again 💥

**Comment by clari182** [comment]
@cesarvarela conflicts resolved!

**Review by cesarvarela — CHANGES_REQUESTED** [review]
something going on:

image

**Review by cesarvarela — APPROVED** [review]

File: site/gatsby-site/src/components/landing/QuickSearch.js [hunk 1]
```diff
@@ -1,46 +1,41 @@
...
-      content: '${(props) => props.t('Search 1600+ AI harm reports')}';
-    }
-    @media (min-width: 500px) {
-      content: '${(props) => props.t('Search over 1600 reports of AI harms')}';
-    }
-  }
-  &.has-input {
-    :after {
-      content: '' !important;
+  const [searchPlaceholder, setSearchPlaceholder] = useState('Search reports');
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/landing/QuickSearch.js:15]: Can you leve this empty?

**Review by cesarvarela — CHANGES_REQUESTED** [review]
The thing with the absolute position of the placeholder was to prevent the "search reports" text from flashing when loading the page. 

Maybe leaving it empty is an option 🤷

**Comment by cesarvarela** [comment]
woot conflicts

**Review by cesarvarela — APPROVED** [review]

**Pull request merged**